app/Services/EntityService.php

```diff
@@ -657,7 +657,14 @@ public function makeNewMentionEntity(MiscModel $model, string $name)
         $model->forceSavedObserver = true;
         $model->is_private = $defaultPrivate;
         $model->save();
-
+        if ($model->entity->type() != 'tag') {
+            $tags = \App\Models\Tag::autoApplied()->get();
+            foreach ($tags as $tag) {
+                if ($tag && $tag->entity) {
+                    $model->entity->tags()->attach($tag->id);
```

Comment:
If we're creating 5 new entities with [new:xxx] syntax and have 5 tags that auto-apply, this will generate 25 extra queries! We can 
a) only get the list of valid tag ids once and save it in an array in the EntityService
b) add all the tags ad once `attach()` accepts an array :)